workspace_client.go

```diff
@@ -367,7 +367,52 @@ type WorkspaceClient struct {
 	ModelRegistry *ml.ModelRegistryAPI
 
 	// Permissions API are used to create read, write, edit, update and manage
-	// access for various users on different objects and endpoints.
+	// access for various users on different objects and endpoints:
+	//
+	// * **[Cluster permissions](clusters)** — Manage which users can manage,
```

Comment:
we have to use different syntax for cross-service referencing or links. See `Deep linking: methods, classes, and external links` from http://go/openapi/spec. 

can you also add a linter to prevent these from being added in universe?

Reply:
Yes, will add a linter there.

Reply:
![Screenshot 2023-08-03 at 13 28 53](https://github.com/databricks/databricks-sdk-go/assets/1850319/cedf4d69-8c95-4ab9-a4fc-01fcf0689634)
That style of link also does not seem to be handled appropriately?

Reply:
Filed a ticket with @arusanov to add linting